### DISCUSSION TITLE
`Revert to default` removes `DefaultSongProgressBar`

### DISCUSSION THREAD

**Discussion by 25pwn** [ppy/osu#20550] [Q&A]
### Type

Game behaviour

### Bug description

It used to be that `DefaultSongProgressBar` was not shown for newly imported skins and `Revert to default` removed it for skins that did have it. Currently, it is shown but `Revert to default` still removes it.

### Screenshots or videos

Newly imported:
![Screenshot_20220911_144559](https://user-images.githubusercontent.com/32491885/189514351-3e87eb8b-9a22-4900-b08f-6cc9c19703f5.png)
After `Revert to default`
![Screenshot_20220911_144607](https://user-images.githubusercontent.com/32491885/189514353-90f0dcb4-97be-45c2-8a70-0a7789e47c19.png)

### Version

2022.9.10-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/9541867/database.log)
[network.log](https://github.com/ppy/osu/files/9541868/network.log)
[performance.log](https://github.com/ppy/osu/files/9541869/performance.log)
[runtime.log](https://github.com/ppy/osu/files/9541870/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9541871/updater.log)

**Comment by peppy** [comment]
Can you confirm that your concern is that when you import a legacy skin from scratch, it has the progress bar at the bottom?

That doesn't seem to be the case for me. Can you provide a skin showing this issue?

**Comment by 25pwn** [comment]
(Sorry for the late reply, I never get notifications for some reason)

As I noted above, in the past(not anymore), newly imported skins did not have a progress bar. Currently, newly imported skins do have a progress bar.

However, my concern is that those skins imported in the past that did not have a progress bar remain without one, and(both in the past and currently,) `Revert to default` removes the progress bar.

**Comment by peppy** [comment]
Newly imported skins don't have a progress bar. I think you might be mistaken? You will need to add it back manually.

**Comment by 25pwn** [comment]
@peppy Again, you are misunderstanding this:

Some past versions: Newly imported skins **did not** have a progress bar. `Revert to default` removed the progress bar if a progress bar was present.

Latest version: Newly imported skins **do** have a progress bar.  `Revert to default` removes the progress bar if a progress bar is present..

**Reply by peppy** [reply]
Please make a video showing what you're seeing.
